src/DomHandler/Wsdl/Wsdl.php

```diff
@@ -78,7 +78,12 @@ public function getElementsByName($name, $includeExternals = false)
             $name,
         ), $includeExternals);
     }
+
```

Comment:
Indeed but [line 81](https://github.com/WsdlToPhp/PackageGenerator/pull/70/files/8ed83e77bf3b553721664e028173a34051e73af9#diff-f2a43b16493644355096c1c7656a8984L80) is not the end of the file :wink: